src/ignitus-Common-Auth-Student-Prof/Components/CommonLoginUI.tsx

```diff
@@ -1,10 +1,10 @@
 import React from 'react';
-import { Link } from 'react-router-dom';
-import { withErrorBoundary } from '../../ignitus-Internals';
-import * as t from './Constants';
+import {Link} from 'react-router-dom';
+import {withErrorBoundary} from '../../ignitus-Internals';
+import {studentAuth, professorAuth} from './constants';
```

Comment:
hey, @steric85  any specific reason to remove `import * as t from './Constants';`, it should be fine to keep it there, to do migration you just have to define types, you don't have to change the way you are importing them in this case.

Reply:
In my opinion destructuring in with ` { }  ` is a good approach.

Reply:
cool, so `destructuring`. in you opinion is good approach everytime while importing or if and only if there are lets say 2-3 constants that needs import, or lets say if there are 6 constant then will you prefer importing 6 of them in using `destructuring`?

Reply:
Okay so @steric85 should change that statement. Thanx @divyanshu-rawat  for clearing it.

Reply:
thanks @debck , we think @steric85, you are on right direction keep working on it :)

Reply:
Sure @divyanshu-rawat . I'll revert the import change.